Soomla/Assets/Plugins/Soomla/Profile/Domain/UserProfile.cs

```diff
@@ -80,48 +80,48 @@ protected UserProfile(Provider provider, string profileId, string username)
 			this.Username = JSONObject.DecodeJsString(jsonUP[PJSONConsts.UP_USERNAME].str);
 			this.ProfileId = JSONObject.DecodeJsString(jsonUP[PJSONConsts.UP_PROFILEID].str);
 
-			if (jsonUP[PJSONConsts.UP_FIRSTNAME]) {
```

Comment:
@vedi, because JSONObject can be not-null, but contains NULL-string (returned by `.str` method). Might be better is to add `jsonUP[blahblah].str != null`, but I'm thinking that this method is more versatile.